src/components/forms/NumberField/NumberField.stories.mdx

```diff
@@ -26,7 +27,14 @@ const generalTest = async canvasElement => {
 <Meta
   title="Pure React Components / Forms / NumberField"
   component={NumberField}
+  decorators={[FormikDecorator]}
```

Comment:
we can specify the decorators at the meta level instead of repeating it for every component, bit more DRY as it applies to every story anyway.